### PR TITLE
[FIX] Fixed compilation when libjpeg is missing 

### PR DESCRIPTION
**Pull request by ferencIOS**
- [x] Checked coding [style](https://github.com/pdfmm/pdfmm/blob/master/CODING-STYLE.md)
- [x] Accept to license the code under the terms of the [LGPL 2.0](https://spdx.org/licenses/LGPL-2.0-or-later.html) or later
- [x] Accept to license the code under the terms of the [MPL 2.0](https://spdx.org/licenses/MPL-2.0)

### CODE REVIEW

File: src/pdfmm/private/JpegCommon.cpp [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#ifdef PDFMM_HAVE_JPEG_LIB
```
Comment (ceztko) [inline src/pdfmm/private/JpegCommon.cpp:1]: Please, don't add such ifdefs to disable whole files but check the source exclusions in CMakeLists for correct working:

[if(NOT PDFMM_HAVE_JPEG_LIB)](https://github.com/pdfmm/pdfmm/blob/27fa0d84d660cb85ec8b7459d96112f3c2e555c0/src/pdfmm/private/CMakeLists.txt#L6)

After you have checked this please resend without the ifdefs in `JpegCommon.h`, `JpegCommon.cpp` .

Comment (ferencIOS) [reply src/pdfmm/private/JpegCommon.cpp]: you are right, now it's clear!

**Comment by ceztko** [comment]
Thank you. Please squash the commits in a single one.

**Comment by ceztko** [comment]
Thank you. I had a compilation error in the revision you rebased to but I merged the change anyway since I already fixed it.

**Comment by ceztko** [comment]
By the way this project will move back to [PoDoFo](https://github.com/podofo/podofo) soon. I'm closing ability to create PRs.

**Pull request merged**